hudi-client/hudi-spark-client/pom.xml

```diff
@@ -127,6 +127,14 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- Other Utils -->
```

Comment:
can we pull this into parent pom, introduce a property for the version and just use this without the `<version>` tag here. just like other deps/